dotnet/src/VectorDataIntegrationTests/SqlServerIntegrationTests/SqlServerCommandBuilderTests.cs

```diff
@@ -0,0 +1,371 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.VectorData;
+using Microsoft.SemanticKernel.Connectors.SqlServer;
+using Xunit;
+
+namespace SqlServerIntegrationTests;
+
+public class SqlServerCommandBuilderTests
```

Comment:
See #10464 - I don't believe we should be writing unit tests on SQL generated by internal components of the provider; we should instead lean on end-to-end, reliable integration tests. But this is all OK for now (other connectors do the same), we'll look at all this later.